src/core/components/providers/markdown.jsx

```diff
@@ -27,7 +27,7 @@ function Markdown({ source, className = "" }) {
       // If the source text is not Markdown,
       // let's save some time and just render it.
       return <div className="markdown">
-        {source}
+        <p>{source}</p>
```

Comment:
**Question**: what's this for?

Reply:
Again here I'm trying to address an inconsistency in html structure between plainText and markdown returned by "Remarkable" - which seems to be wrapped in a `<p>` tag.

Reply:
indeed, looks like Remarkable does always wrap with a paragraph: https://github.com/jonschlinkert/remarkable/issues/282

all good here!